dll/apisets/api-ms-win-crt-convert-l1-1-0.spec

```diff
@@ -7,13 +7,13 @@
 @ stub _atof_l
 @ stub _atoflt
 @ stub _atoflt_l
-@ stdcall _atoi64() msvcrt._atoi64
-@ stub _atoi64_l
+@ cdecl _atoi64() msvcrt._atoi64
+@ stub -ret64 _atoi64_l
```

Comment:
Does this flag have any effect on stubs?

Reply:
No, it's only useful for wine relay tracing, which we don't support.

Reply:
actually, we do ;)
https://reactos.org/wiki/Debugging#Log_api_calls_to_a_specific_ReactOS_dll